src/app/components/page.component.ts

```diff
@@ -194,9 +194,13 @@ export abstract class PageComponent implements OnInit, OnDestroy {
         let message = null;
 
         if (error.error.status === 409) {
-           message =  new SnackbarErrorAction('VERSION.MESSAGE.ERROR.CONFLICT');
+           message =  new SnackbarErrorAction('APP.MESSAGES.UPLOAD.ERROR.CONFLICT');
         }
 
+        if (error.error.status === 500) {
+            message =  new SnackbarErrorAction('APP.MESSAGES.UPLOAD.ERROR.500');
+         }
```

Comment:
can we also have some generic fallback message in case there was an error with some exotic code that we don't check for?